template.yml

```diff
@@ -508,6 +509,67 @@ Resources:
       Principal: events.amazonaws.com
       SourceArn: !GetAtt STMAnalyzeDailyStopsDataTrigger.Arn
 
+
+####################### ANALYZE SEGMENTS #######################
+  STMAnalyzeDailyStopsData:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      Description: "Analyze the time we gain back between each bus station"
+      Timeout: 840 # Timeout in seconds
+      MemorySize: 8192
+      CodeUri: STM_Services/STM_Analyse_Segments/
+      Handler: main.lambda_handler
+      Runtime: python3.9
+      Architectures:
+        - x86_64
+      Environment:
+          Variables:
+            API_URL_STM_TRIP: !Ref ApiUrlStmTrip
+            API_URL_STM_VEHICLE: !Ref ApiUrlStmVehicle
+            API_KEY_STM: !Ref ApiKeyStm
+
+  STMAnalyzeSegmentsDataVersion:
+    Condition: SetSTMAnalyzeSegmentsData
+    Type: AWS::Lambda::Version
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      FunctionName: !Ref SetSTMAnalyzeSegmentsData
+
+  SetSTMAnalyzeSegmentsDataAlias:
+    Condition: SetSTMAnalyzeSegmentsData
+    Type: AWS::Lambda::Alias
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      FunctionName: !Ref SetSTMAnalyzeSegmentsData
+      FunctionVersion: !GetAtt SetSTMAnalyzeSegmentsDataVersion.Version
+      Name: Live
+  
+  SetSTMAnalyzeSegmentsDataTrigger:
+    Type: AWS::Events::Rule
+    Properties:
+      Description: "Calls SetSTMAnalyzeSegmentsData every day at 3:00 AM"
+      ScheduleExpression: cron(0 0 3 1/1 * ? *)
```

Comment:
Why use 1/1 here instead of * when it achieves the same result?

Reply:
Also too many parameters in your cron

Reply:
fixed, changed to (0 3 * * *)